tests/unit/setup-ember-onerror-test.js

```diff
@@ -5,7 +5,7 @@ import { setupOnerror, resetOnerror } from '@ember/test-helpers';
 
 module('setupOnerror', function(hooks) {
   hooks.afterEach(function() {
-    resetOnerror();
+    Ember.onerror = undefined;
```

Comment:
Calling `resetOnerror` is intended to reset the error state to the very first state that was stashed when the module was required. This shouldn't be reset to `undefined` explicitly, but rather to the original error state as defined [here](https://github.com/emberjs/ember-test-helpers/blob/master/addon-test-support/@ember/test-helpers/setup-onerror.ts#L3). Other than this, added the test case seems fine.